src/oauth-provider.ts

```diff
@@ -475,6 +488,11 @@ export interface Grant {
    */
   createdAt: number;
 
+  /**
+   * Unix timestamp when the refresh token expires (if TTL is configured)
+   */
+  refreshTokenExpiresAt?: number;
```

Comment:
When the refresh token expires, the grant itself effectively expires, so I think this can just be called `expiresAt`.

Reply:
Agreed, that is much cleaner